tests/problem-set-tests.md

```diff
@@ -19,5 +65,55 @@ const mystery = (text) => {
 };
 ```
 ##### !end-placeholder
+##### !hint
+
+Copy and paste the tests into a file so you can look at these tests side-by-side.
+
+##### !end-hint
+##### !hint
+
+There are many ways to implement `mystery`. Some implementations use one or more of these:
+
+- `forEach` loops
+- Using an object to map a key to a boolean (like a hash table)
+- Using regular expressions
+- `for ... of` loops
+
+Note: You cannot use `return` to exit a `forEach` loop. If you need to break out of a `forEach` loop, try refactoring to a `for ... of` loop.
+
+##### !end-hint
+##### !hint
+
+The responsibility of the function `mystery` is to return whether the input `text` is a [pangram](https://en.wikipedia.org/wiki/Pangram).
+
+##### !end-hint
+##### !explanation
+
+An example of a working implementation:
+
+```js
+const mystery = (text) => {
+  const requiredLetters = 'abcdefghijklmnopqrstuvwxyz'.split('');
+  const letterPresenceMap = {};
+
+  requiredLetters.forEach((letter) => {
```

Comment:
I think for C16, I'll have to do a deeper look at replacing `forEach`, since this isn't my first time hearing about how terrible it is hahaha

For now let's keep it as a way to apply anonymous functions! Thanks for the thought ~